docs/advanced/naming.md

```diff
@@ -1,160 +1,198 @@
-- A common thing in anime releases is Specials/OVAs numbered as `0.5`, e.g. `Anime Name - 18.5`. This is not recommended at all.
-- `0.5` Episodes are not supported by a single piece of software or database but they usually make it easier for the person watching to know where it chronologically lands in the show. You may or may not include this in the filename.
-- `S00EXX` isn't very user friendly, because it doesn't really tell much on its own but it guarantees automatic parsing and correct identification. You must include this in the filename. Refer to [thetvdb](https://thetvdb.com/) to find this for your episode.
-- Including the Episode title is highly recommended in this case as it makes identification significantly easier and understandable for anyone who downloads it. Neither `S00EXX` or `0.5` provides this information at a glance which is why the episode title is really helpful in this case. Refer to either [thetvdb](https://thetvdb.com/) or [MAL](https://myanimelist.net/) to find the episode name.
-- You may include the absolute number of the episode as long as it's not a decimal episode that lands between any two existing ones. Optional and really just down to preference. It's much better than decimal episode numbering but still not ideal.
-- This means we need a name that is easily readable, parsable, and identifiable. This is achieved by using a combo of the above.
+![Releases using bad naming schemes](/static/advanced/naming/releases-bad-naming.png)
 
-```
-Anime Name - S00E04 - Title of the Episode
-```
+These naming schemes are **bad** as they:
 
-- This is an example of a well named special episode. `Anime Name` tells us the name of the anime, `S00E04` tells us the episode number in the `SXXEXX` standard which makes it easily identifiable, and `Title of the Episode` makes it significantly more human readable.
+Do not include the [type of release](/guides/quality/#types-of-releases) and [where it is sourced from](/guides/quality/#blu-ray-vs-web)
+:   - File 2 does not list the video codec/resolution, audio codec, "Blu-ray/WEB" source, dual audio, etc.
 
-```
-Anime Name - 18.5 - S00E05 - Title of the Episode
-```
+May be too long in length
+:   - File 2 includes episode chapters in the file name, which results in a long file name
 
-- Another good example, this tells us the name of the anime, where it lands chronologically, what episode it is with reference to databases and the name of the episode to aid with identifying the episode without much effort.
+Use an invalid naming scheme
+:   - File 1 separates the season and episode number (`S2 - 08`). *As a result, some programs may incorrectly parse this as `S01E08`*
```

Comment:
you can remove this entirely, most of them handle `S2 - 08` more often than not. Replace it with an example that uses absolute numbering and recommend SxxEyy instead